### PR TITLE
Pass the Start and End of the EA search

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#74974](http://central.tri.be/issues/74974)

### CODE REVIEW

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 1]
```diff
@@ -609,7 +609,16 @@ public function queue_import( $args = array() ) {
...
+
+		// For now only apply this to the URL type
+		if ( 'url' === $this->type ) {
+			$data = array(
+				'start' => $this->meta['start'],
+				'end' => $this->meta['end'],
+			);
+		}
+
+		return $aggregator->api( 'import' )->get( $this->meta['import_id'], $data );
```
Comment (barryhughes) [inline src/Tribe/Aggregator/Record/Abstract.php]: _Looks_ like `$data` may not be defined at this point (depending on the result of the above conditional). Worth defining it as an empty array, some point above the `if` block?

Comment (bordoni) [inline src/Tribe/Aggregator/Record/Abstract.php]: Ohh very good point.

**Comment by bordoni** [comment]
It's ready for Code Review again

**Comment by borkweb** [comment]
:+1:

**Pull request merged**